synapse/lib/scrape.py

```diff
@@ -24,6 +24,15 @@
 
 logger = logging.getLogger(__name__)
 
+urilist = list(s_data.get('iana.uris'))
+urilist.extend([
+    'ftps',
+    'tcp'
+])
+urilist.sort(key=lambda x: len(x))
+urilist.reverse()
+uricat = '|'.join(urilist)
```

Comment:
Looks like this was intended to be added to the regex but doesn't get used, can probably remove it.

Reply:
Could also probably make urilist a set rather than list